src/configs/git_branch.rs

```diff
@@ -0,0 +1,27 @@
+use crate::config::{ModuleConfig, RootModuleConfig, SegmentConfig};
+
+use ansi_term::{Color, Style};
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct GitBranchConfig<'a> {
+    pub symbol: SegmentConfig<'a>,
+    pub truncation_length: i64,
+    pub truncation_symbol: &'a str,
+    pub branch_name: SegmentConfig<'a>,
```

Comment:
This segment was named `name` in the original file, but I thought it was a little ambiguous if it is going to be customizeable from the config file. Should I change it back? Or should we go all the way and even change the name of the segment (line 42 of modules/git_branch.rs) to "branch_name"?

Reply:
Ok for both as these configs are currently undocumented and are not officially available for users. I've opened an issue for that #472.